src/redisraft.h

```diff
@@ -839,8 +910,14 @@ const CommandSpec *CommandSpecGet(const RedisModuleString *cmd);
 void handleSort(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 
 /* multi.c */
-void MultiInitClientState(RedisRaftCtx *rr);
-uint64_t MultiClientStateCount(RedisRaftCtx *rr);
-void MultiFreeClientState(RedisRaftCtx *rr, unsigned long long client_id);
+void MultiClientStateReset(ClientState *clientState);
```

Comment:
```suggestion
void MultiClientStateReset(ClientState *client_state);
```
Please use snake case for variable names